CHANGELOG.md

```diff
@@ -111,10 +111,11 @@ All notable, unreleased changes to this project will be documented in this file.
 - Fix fetching staff user without `manage_users` permission - #4835 by @fowczarek
 - Ensure that a GraphQL query is a string - #4836 by @nix010
 - Add ability to configure the password reset link - #4863 by @fowczarek
+- Dropped products prefetch when resolving collections and categories - #5291 by @NyanKiyoshi
```

Comment:
I don't think that's an informative changelog entry. As a user, I'd rather read something like "fixed a performance issue where Saleor would sometimes…"